DANS-KNAW/easy-deposit-api
# EASY0-1794 POST upload should not replace existing files

Fixes EASY-1794 POST upload should not replace existing files 

#### When applied it will
* [x] ~~test concurrent uploads: POST while PUT is busy moving files
  though the specs say that the result is undefined.~~
* [ ] https://github.com/janvanmansum/easy-deposit-api/blob/26413829076ee21e3cc1dddb36612932abee7f36/docs/api/api.yml#L646-L650
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* on deasy: create a directory in `/var/opt/dans.knaw.nl/deposit/stage-upload/`
* redeploy or restart
* the service will not start, the last log line explains:

        ERROR Pending uploads were probably interrupted. See logs lines with 'POST /deposit/{id}/file/{dir_path}'. Please remove their directories from the staging area: /var/opt/dans.knaw.nl/deposit/stage-upload

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
